app/javascript/tests/layout/Header.test.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Nav, Navbar } from 'react-bootstrap';
+import Header from '../../components/layout/Header';
+import { mockUser1 } from '../mocks/mockUsers';
+
+const helpLinks = {
+  user_guides: 'https://www.sara.org/user-guides',
+  user_forum: 'https://www.sara.org/user-forum',
+  contact_us: 'https://www.sara.org/contact-us',
+};
+
+const bannerMessage = 'This is a test banner message.';
+const version = '123';
+
+/**
+ * If additionalProps undefiend, the Header component is rendered with the default props included below
+ * If any props are defined in additional props, they overwrite whatever the default value was set as
+ */
+function getWrapper(additionalProps) {
+  return shallow(<Header report_mode={false} version={version} show_demo_warning_background={false} banner_message="" current_user={mockUser1} help_links={helpLinks} {...additionalProps} />);
+}
+
+function getMainNav(wrapper) {
+  return wrapper.find(Navbar).find({ bg: 'primary' });
+}
+
+describe('Header', () => {
+  it('Properly renders all main components', () => {
+    mockUser1.role = 'fake_role';
+    const wrapper = getWrapper({ current_user: mockUser1 });
+    const mainNav = getMainNav(wrapper);
+    expect(mainNav.exists()).toBe(true);
+    expect(mainNav.find(Navbar.Brand).exists()).toBe(true);
+    expect(mainNav.find(Navbar.Brand).text()).toContain(`Sara Alert${version}`);
+    expect(mainNav.find(Navbar.Text).exists()).toBe(true);
+    expect(mainNav.find(Navbar.Text).text()).toContain(`${mockUser1.email} (Fake Role)`);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/users/sign_out` })
+        .exists()
+    ).toBe(true);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/users/sign_out` })
+        .text()
+    ).toEqual('Logout');
+    expect(mainNav.find(Nav.Link).find({ id: 'helpMenuButton' }).exists()).toBe(true);
+    expect(mainNav.find(Nav.Link).children().find('i').exists()).toBe(true);
+    expect(mainNav.find('.dropdown-menu').exists()).toBe(true);
+    expect(mainNav.find({ children: 'Enroller Dashboard' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Monitoring Dashboards' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Admin Panel' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Analytics' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'API' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Jobs' }).exists()).toBe(false);
+    expect(wrapper.find(Navbar).find({ bg: 'warning' }).exists()).toBe(false);
+  });
+
+  it('Properly renders Enroller Dashboard tab', () => {
```

Comment:
```suggestion
  it('Shows "Enroller Dashboard" tab when user. can_see_enroller_dashboard_tab', () => {
```

Reply:
same with the following tabs

Reply:
[add8fdf](https://github.com/SaraAlert/SaraAlert/pull/1243/commits/add8fdf480050ab21714edbf52b4d49bcddc348a)